configs/components/echam/echam.yaml

```diff
@@ -548,6 +593,7 @@ choose_wiso_code:
                                 add_streams:
                                         - wiso
                                         - accw_wiso
+                                        # TODO: these two are missing in release
                                         - sf_wiso
                                         - rad
```

Comment:
And that is as it should be. It's one of the latest fixes. It will be solved when merging from release into this branch.